pixeltable/utils/object_stores.py

```diff
@@ -472,6 +459,16 @@ def delete(cls, dest: str | None, tbl_id: UUID, tbl_version: int | None = None)
         store = cls.get_store(dest, False)
         return store.delete(tbl_id, tbl_version)
 
+    @classmethod
+    def count_default_input_dest(cls, tbl_id: UUID, tbl_version: int | None = None) -> int:
```

Comment:
Roll this into count(default_input=True)/count(default_output=True) to cut down on overall complexity of this already complex class hierarchy?